--- ./fltmpS/slider.cxx	2020-02-25 18:23:18.359319898 -0700
+++ ../src/slider.cxx	2020-02-24 19:53:40.069974807 -0700
@@ -1,20 +1,72 @@
+// slider.cxx
 //
-// "$Id$"
+// "$Id: Fl_Slider.cxx 11536 2016-04-05 21:15:45Z AlbrechtS $"
 //
 // Slider widget for the Fast Light Tool Kit (FLTK).
 //
+// Copyright 2017-2018 The fltkal authors
 // Copyright 1998-2015 by Bill Spitzak and others.
 //
-// This library is free software. Distribution and use rights are outlined in
-// the file "COPYING" which should have been included with this file.  If this
-// file is missing or damaged, see the license at:
+//                              FLTK License
+//                            December 11, 2001
 //
-//     http://www.fltk.org/COPYING.php
+// The FLTK library and included programs are provided under the terms
+// of the GNU Library General Public License (LGPL) with the following
+// exceptions:
 //
-// Please report all bugs and problems on the following page:
+//     1. Modifications to the FLTK configure script, config
+//        header file, and makefiles by themselves to support
+//        a specific platform do not constitute a modified or
+//        derivative work.
+// 
+//       The authors do request that such modifications be
+//       contributed to the FLTK project - send all contributions
+//       through the "Software Trouble Report" on the following page:
 //
 //     http://www.fltk.org/str.php
 //
+//     2. Widgets that are subclassed from FLTK widgets do not
+//        constitute a derivative work.
+// 
+//     3. Static linking of applications and widgets to the
+//        FLTK library does not constitute a derivative work
+//        and does not require the author to provide source
+//        code for the application or widget, use the shared
+//        FLTK libraries, or link their applications or
+//        widgets against a user-supplied version of FLTK.
+// 
+//        If you link the application or widget to a modified
+//        version of FLTK, then the changes to FLTK must be
+//        provided under the terms of the LGPL in sections
+//        1, 2, and 4.
+// 
+//     4. You do not have to provide a copy of the FLTK license
+//        with programs that are linked to the FLTK library, nor
+//        do you have to identify the FLTK license in your
+//        program or documentation as required by section 6
+//        of the LGPL.
+// 
+//        However, programs must still identify their use of FLTK.
+//        The following example statement can be included in user
+//        documentation to satisfy this requirement:
+// 
+//            [program/widget] is based in part on the work of
+//            the FLTK project (http://www.fltk.org).
+// 
+//     This library is free software; you can redistribute it and/or
+//     modify it under the terms of the GNU Library General Public
+//     License as published by the Free Software Foundation; either
+//     version 2 of the License, or (at your option) any later version.
+// 
+//     This library is distributed in the hope that it will be useful,
+//     but WITHOUT ANY WARRANTY; without even the implied warranty of
+//     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+//     Library General Public License for more details.
+// 
+//     You should have received a copy of the GNU Library General Public
+//     License along with FLTK.  If not, see <http://www.gnu.org/licenses/>.
+//
+//
 
 #include <fl/fl.h>
 #include <fl/slider.h>
@@ -24,7 +76,7 @@
 #include <fl/slhznice.h>
 #include <fl/slnice.h>
 #include <fl/fl_draw.h>
-#include <fl_math.h>
+#include <fl/fl_math.h>
 #include "flstring.h"
 
 
@@ -398,5 +450,5 @@
 
 
 //
-// End of "$Id$".
+// End of "$Id: Fl_Slider.cxx 11536 2016-04-05 21:15:45Z AlbrechtS $".
 //
